packages/ui/src/Modal.tsx

```diff
@@ -88,6 +88,9 @@ const Content = styled(CenterLayout)`
     height: auto;
   }
 `
+const ChildrenWrapper = styled.div`
+  min-width: 450px;
```

Comment:
maybe change this to `width: 100%;max-width: 450px` ? so that it can be responsive to some point.

Reply:
@aleksaToljic in provided design modals are 450px width, if I set it to 100% it will be shorter.
Maybe to set min-width/width for mobile?
![image](https://user-images.githubusercontent.com/42462482/178969419-270aaaac-0979-4d78-8d06-276c0afdbde5.png)

Reply:
@aleksaToljic with this changes modal width will be shorter then it should be (in design it is 450px)
![image](https://user-images.githubusercontent.com/42462482/178973808-4aeca690-ba9c-4afa-a913-9f808f0dbfb2.png)

Maybe to set width: 100% for mobile?

Reply:
![image](https://user-images.githubusercontent.com/22115870/179188898-3d6a3d0a-8b01-4cc5-9d46-4375186d7b45.png)
these things are needed that are in "style" attribute